safari.sh

```diff
@@ -0,0 +1,5 @@
+rm -r build/safari
```

Comment:
```suggestion
#!/bin/bash

set -euo pipefail

rm -rf build/safari
```

Lets set the shell + enable script mode + use f to stop freak out when dir exists

http://redsymbol.net/articles/unofficial-bash-strict-mode/